### PR TITLE
translate 6 PRs for functions and operators

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
    - https://github.com/pingcap/docs/pull/16847
    - https://github.com/pingcap/docs/pull/16851
    - https://github.com/pingcap/docs/pull/16852
    - https://github.com/pingcap/docs/pull/16854
    - https://github.com/pingcap/docs/pull/16857
    - https://github.com/pingcap/docs/pull/16859
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/bit-functions-and-operators.md [hunk 1]
```diff
@@ -19,3 +19,14 @@ TiDB 支持使用 MySQL 5.7 中提供的所有[位函数和操作符](https://de
 | [^](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_bitwise-xor) | 位亦或 |
 | [<<](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_left-shift) | 左移 |
 | [>>](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_right-shift) | 右移 |
+
+## MySQL 兼容性
+
+在处理位函数和操作符时，MySQL 8.0 与早期版本的 MySQL 之间存在一些差异。TiDB 旨在遵循 MySQL 8.0 的行为。
```
Comment (qiancai) [inline functions-and-operators/bit-functions-and-operators.md:25]:
```suggestion
在处理位函数和操作符时，MySQL 8.0 与之前版本的 MySQL 之间存在一些差异。TiDB 旨在遵循 MySQL 8.0 的行为。
```

File: functions-and-operators/bit-functions-and-operators.md [hunk 2]
```diff
@@ -19,3 +19,14 @@ TiDB 支持使用 MySQL 5.7 中提供的所有[位函数和操作符](https://de
...
+
+## 已知问题
+
+在以下情况中，TiDB 中的查询结果与 MySQL 5.7 相同，但与 MySQL 8.0 不同。
+
+- 二进制参数的位操作。更多信息，请参见 [#30637](https://github.com/pingcap/tidb/issues/30637)。
+- `BIT_COUNT()` 函数的结果。更多信息，请参见 [#44621](https://github.com/pingcap/tidb/issues/44621)。
```
Comment (qiancai) [inline functions-and-operators/bit-functions-and-operators.md:32]:
```suggestion
- 二进制参数的位操作。更多信息，请参考 [#30637](https://github.com/pingcap/tidb/issues/30637)。
- `BIT_COUNT()` 函数的结果。更多信息，请参考 [#44621](https://github.com/pingcap/tidb/issues/44621)。
```

File: functions-and-operators/json-functions.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 title: JSON 函数
 aliases: ['/docs-cn/dev/functions-and-operators/json-functions/','/docs-cn/dev/reference/sql/functions-and-operators/json-functions/']
-summary: TiDB 支持 MySQL 5.7 GA 版本发布的大多数 JSON 函数，包括创建、搜索、修改、返回属性和效用函数。这些函数可用于处理 JSON 值，如创建 JSON 文档、搜索 JSON 文档中的值、修改 JSON 文档、返回 JSON 值属性和聚合函数。此外，还有其他相关函数可供参考。
+summary: TiDB 支持 MySQL 8.0 中提供的大部分 JSON 函数。
 ---
 
 # JSON 函数
 
-TiDB 支持 MySQL 5.7 GA 版本发布的大多数 JSON 函数。
+TiDB 支持 MySQL 8.0 中提供的大部分 [JSON 函数]((https://dev.mysql.com/doc/refman/8.0/en/json-functions.html))。
```
Comment (hfxsd) [inline functions-and-operators/json-functions.md:9]:
```suggestion
TiDB 支持 MySQL 8.0 中提供的大部分 [JSON 函数](https://dev.mysql.com/doc/refman/8.0/en/json-functions.html)。
```

File: functions-and-operators/json-functions.md [hunk 4]
```diff
@@ -35,18 +35,16 @@ TiDB 支持 MySQL 5.7 GA 版本发布的大多数 JSON 函数。
 | 函数        | 功能描述 |
 | --------------------------------- | ----------- |
 | [JSON_APPEND(json_doc, path, value)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-append) | `JSON_ARRAY_APPEND` 的别名 |
-| [JSON_ARRAY_APPEND(json_doc, path, value)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-append) | 将值追加到指定路径的 JSON 数组的末尾 |
-| [JSON_ARRAY_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-insert) | 将数组插入 JSON 文档，并返回修改后的文档 |
+| [JSON_ARRAY_APPEND(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-append) | 将值添加到 JSON 文档指定数组的末尾，并返回添加结果 |
+| [JSON_ARRAY_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-insert) | 将值插入到 JSON 文档中的指定位置并返回结果。 |
```
Comment (hfxsd) [inline functions-and-operators/json-functions.md:39]:
```suggestion
| [JSON_ARRAY_INSERT(json_doc, path, val[, path, val] ...)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-array-insert) | 将值插入到 JSON 文档中的指定位置并返回结果 |
```

File: functions-and-operators/json-functions.md [hunk 5]
```diff
@@ -35,18 +35,16 @@ TiDB 支持 MySQL 5.7 GA 版本发布的大多数 JSON 函数。
 | 函数        | 功能描述 |
 | --------------------------------- | ----------- |
 | [JSON_APPEND(json_doc, path, value)](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-append) | `JSON_ARRAY_APPEND` 的别名 |
```
Comment (hfxsd) [inline functions-and-operators/json-functions.md:37]: 此处括号里的内容要改成 (json_doc, path, val[, path, val] ...) 吗？

**Review by hfxsd — APPROVED** [review]

File: functions-and-operators/json-functions.md (hunk 5 above)
Comment (qiancai) [reply functions-and-operators/json-functions.md:37]: 这个 PR 只是重新排序了这个表格里的行，没有改每行里的内容，所以不需要改。

https://github.com/pingcap/docs/pull/16859/files#r1580354189

**Pull request merged**